flow/project.py

```diff
@@ -2668,7 +2672,7 @@ def _generate_operations(self, cmd, jobs, requires=None):
             if requires and set(requires).difference(self.labels(job)):
                 continue
             cmd_ = cmd.format(job=job)
-            yield _JobOperation(name=cmd_.replace(' ', '-'), cmd=cmd_, job=job)
+            yield JobOperation(name=cmd_.replace(' ', '-'), cmd=cmd_, job=job)
```

Comment:
Why are we using `JobOperation` rather than `_JobOperation`?

Reply:
I think I accidentally did this